images/macos/provision/core/android-toolsets.sh

```diff
@@ -17,14 +17,22 @@ function filter_components_by_version {
     done
 }
 
+function get_latest_major_ndk {
```

Comment:
nitpick: this function name is a little confusing for me. Can we rename it like `get_full_ndk_version`?

Reply:
Done.